src/components/layouts/theme/ThemeProvider.tsx

```diff
@@ -13,7 +14,9 @@ const client = createConfig({
 const ClientProviders = ({ children }: PropsWithChildren) => {
   return (
     <ThemeProvider attribute="class" defaultTheme="dark">
-      <WagmiConfig config={client}>{children}</WagmiConfig>
+      <WagmiConfig config={client}>
```

Comment:
not related but why do we have have a wagmi wrapper in this static page?

Reply:
I saw some contract read, not sure why but I’ll check what they do :))

Reply:
<img width="674" alt="Screenshot 2024-06-23 at 5 07 55 PM" src="https://github.com/EveripediaNetwork/braindao-ui/assets/66301634/7902cc8c-d965-4061-950e-bfad3cd2af25">


It's used for the stat pointers. I think we can remove it and use fetch calls since it's use a little potion of the app.